lib/fog/compute/voxel.rb

```diff
@@ -1,23 +1,21 @@
-require 'fog/voxel'
-
 module Fog
   module Compute
     class Voxel < Fog::Service
-      autoload :Image, 'fog/compute/voxel/image'
-      autoload :Images, 'fog/compute/voxel/images'
-      autoload :Server, 'fog/compute/voxel/server'
-      autoload :Servers, 'fog/compute/voxel/servers'
+      autoload :Image, File.expand_path('../voxel/models/image', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.